.github/workflows/CI.yml

```
@@ -0,0 +1,39 @@
+name: CI
+on:
+  push:
+    branches:
```

```suggestion
    branches: "master"
```
Otherwise, this would run the tests on every push to any branch, right? I don’t think that is necessary.